features/basics/partial_test_doubles.feature

```diff
@@ -1,7 +1,7 @@
 Feature: Partial test doubles
 
   A _partial test double_ is an extension of a real object in a system that is instrumented with
-  test-double like behaviour in the context of a test. This technique is very common in Ruby
+  test-double-like behaviour in the context of a test. This technique is very common in Ruby
```

Comment:
Woops, sorry, my bad. But in that case, I should also remove the hyphen in [this line](https://github.com/rspec/rspec-mocks/blob/master/features/basics/partial_test_doubles.feature#L22) (I mistakenly thought that this was the right syntax to use), shouldn't I?